scripts/release/prepare-release-from-npm-commands/update-stable-version-numbers.js

```diff
@@ -152,10 +138,6 @@ const run = async ({cwd, packages, version, ci}, versionsMap) => {
         while (afterContents.indexOf(version) >= 0) {
           afterContents = afterContents.replace(version, newStableVersion);
         }
-        // Replace inline renderer version numbers (e.g. shared/ReactVersion).
-        while (afterContents.indexOf(reactVersion) >= 0) {
-          afterContents = afterContents.replace(reactVersion, newStableVersion);
-        }
```

Comment:
Since we pull all the same versions in this command, this 2nd loop is basically a no-op after the first one.